reana_db/models.py

```diff
@@ -301,3 +302,24 @@ class JobCache(Base, Timestamp):
     result_path = Column(String(1024))
     workspace_hash = Column(String(1024))
     access_times = Column(JSONType)
+
+
+class AuditLogAction(enum.Enum):
+    """Enumeration of audit log actions."""
+
+    request_token = 0
+
+
+class AuditLog(Base, Timestamp):
+    """Audit log table."""
+
+    __tablename__ = 'audit_log'
+
+    id_ = Column(UUIDType, unique=True, primary_key=True,
+                 default=generate_uuid)
+    user_id = Column(UUIDType, ForeignKey('user_.id_'), nullable=False)
+    action = Column(Enum(AuditLogAction), nullable=False)
```

Comment:
In order to be future-friendly, please foresee adding a `details` JSON column, even if we won't use it much yet.  First possible use case: ask for reason to use REANA via a textarea near "Request token" button with leading text going like "Please explain briefly what would you like to use REANA for" that would be showing for non-physics users.  So the action `details` column would store `{"reason": "foo bar"}`.  See also https://github.com/reanahub/reana-db/issues/18#issuecomment-617660607